portfolio/src/main/webapp/style.css

```diff
@@ -62,6 +62,7 @@
 
 .column-subheader {
     color: #375592;
+    font-weight: 550;
```

Comment:
Why 550? I don't think it a valid value.

Reply:
I was hoping that it would make the font weight be in between 500 and 600 but that doesn't seem to be the case. I removed that line.